free-programming-books-tr.md

```diff
@@ -26,18 +26,22 @@
 ### Android
 
 * [Android Geleceği Yazanlar](https://gelecegiyazanlar.turkcell.com.tr/konu/android)
+* [Android Dersleri] (http://umiitkose.com/android/)
 
 
 ### C
 
 * [C Programlama Diline Giriş](http://www1.gantep.edu.tr/~bingul/c/index.php) - [Ahmet Bingül](http://www1.gantep.edu.tr/~bingul)
 * [Programlamaya Giriş Ders Notları](http://web.itu.edu.tr/uyar/programlama) - [H.Turgut Uyar](http://web.itu.edu.tr/uyar)
+* [C Programlama Dersleri](https://www.youtube.com/watch?v=FtaKEn2f2qI&list=PLfzhcDNz4tlXOvEE7z_u3gEqFO3nwIZ5i&index=1)
+* [GNU C Kütüphanesi Basvuru Klavuzu] (http://www.belgeler.org/glibc/glibc.html)
```

Comment:
```suggestion
### C

* [C Programlama Diline Giriş](http://www1.gantep.edu.tr/~bingul/c/index.php) - [Ahmet Bingül](http://www1.gantep.edu.tr/~bingul)
* [GNU C Kütüphanesi Basvuru Klavuzu] (http://www.belgeler.org/glibc/glibc.html)
* [Programlamaya Giriş Ders Notları](http://web.itu.edu.tr/uyar/programlama) - [H.Turgut Uyar](http://web.itu.edu.tr/uyar)
```

- I removed the YouTube video resource and moved the "GNU C..." resource to be in the correct position.